plot_flight_maneuver.py

```diff
@@ -90,23 +174,41 @@ def animate(self,i):
         ax.set_xlabel('X')
         ax.set_ylim3d([self.y[self.INDEX[self.frame]]-1.0, 1.0+self.y[self.INDEX[self.frame]]])
         ax.set_ylabel('Y')
-        ax.set_zlim3d([-self.z[self.INDEX[self.frame]]-1.0, - self.z[self.INDEX[self.frame]]+1.0])
+        ax.set_zlim3d([self.z[self.INDEX[self.frame]]-1.0, self.z[self.INDEX[self.frame]]+1.0])
         x = []
```

Comment:
I think you should add here: ax.invert_zaxis()
Then you just have to set the sign of the z coordinates such that the nose is up and all should be ok